pkg/ddevapp/laravel.go

```diff
@@ -67,8 +71,8 @@ func laravelPostStartAction(app *DdevApp) error {
 	return nil
 }
 
-// laravelConfigOverrideAction overrides php_version for Laravel, requires PHP8.1
+// laravelConfigOverrideAction overrides php_version for Laravel, requires PHP8.2
 func laravelConfigOverrideAction(app *DdevApp) error {
-	app.PHPVersion = nodeps.PHP81
+	app.PHPVersion = nodeps.PHP82
```

Comment:
Same conversation as above. This is fine, but we need to figure out a more general strategy about whether we override things that are already the default

Reply:
I saw that `laravelConfigOverrideAction` was added when Laravel 10 required PHP 8.1.
I decided to keep it because it will be easier to override it in the future.